common/djangoapps/custom_settings/tests/test_helpers.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime
+
+import pytest
+from mock import Mock
+from pytz import UTC
+
+from custom_settings.helpers import get_course_open_date_from_settings, validate_course_open_date
+from openedx.core.djangoapps.content.course_overviews.tests.factories import CourseOverviewFactory
+
+
+@pytest.mark.parametrize(
+    'course_open_date, expected',
```

Comment:
can we alter `expected` to something more elaborative, like `expected_date`?

Reply:
That would be an overkill, besides this is recommended by pytest through their examples [here](https://docs.pytest.org/en/stable/example/parametrize.html) and [here](https://docs.pytest.org/en/stable/parametrize.html).

A more logical reason would be, expected can be more than one value for a single test, it can be a date for the first iteration, an object on the second iteration, and an exception on the third iteration, etc, so a general word like `expected` is reasonable.